frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -94,45 +123,83 @@ function MissionDefinitionPageBody({ missionDefinition, updateMissionDefinition
         return TranslateText('Inspection required every') + ' ' + returnStringArray.join(', ')
     }
 
+    const onEdit = (editType: string) => {
+        return () => {
+            setIsEditDialogOpen(true)
+            setSelectedField(editType)
+        }
+    }
+
     return (
         <>
-            <KeyValuePairDisplay left={TranslateText('Comment')} right={missionDefinition.comment} />
-            <KeyValuePairDisplay
-                left={TranslateText('Inspection frequency')}
-                right={displayInspectionFrequency(missionDefinition.inspectionFrequency)}
-            />
-            <KeyValuePairDisplay left={TranslateText('Area')} right={missionDefinition.area.areaName} />
-            <KeyValuePairDisplay left={TranslateText('Deck')} right={missionDefinition.area.deckName} />
-            <KeyValuePairDisplay left={TranslateText('Plant')} right={missionDefinition.area.plantCode} />
-            <KeyValuePairDisplay left={TranslateText('Installation')} right={missionDefinition.area.installationCode} />
-            <KeyValuePairDisplay
-                left={TranslateText('Mission source')}
-                right={TranslateText(missionDefinition.sourceType)}
-            />
-            <Button
+            <StyledFormContainer>
+                <KeyValuePairDisplay
+                    left={TranslateText('Name')}
+                    right={missionDefinition.name}
+                    onEdit={onEdit('name')}
+                />
+                <KeyValuePairDisplay left={TranslateText('Area')} right={missionDefinition.area.areaName} />
+                <KeyValuePairDisplay left={TranslateText('Deck')} right={missionDefinition.area.deckName} />
+                <KeyValuePairDisplay left={TranslateText('Plant')} right={missionDefinition.area.plantCode} />
```

Comment:
Should we use plantName instead of plantCode?

Reply:
This wasn't in the backend models, but I've added it now